rest/oidc_api.go

```diff
@@ -118,6 +149,26 @@ func (h *handler) handleOIDCCallback() error {
 		return base.HTTPErrorf(http.StatusBadRequest, "Unable to identify provider for callback request")
 	}
 
+	// Validate state parameter to prevent cross-site request forgery (CSRF) when callback state is enabled.
+	if !provider.DisableCallbackState {
+		stateCookie, err := h.rq.Cookie(stateCookieName)
+
+		if err != nil && err != http.ErrNoCookie {
+			base.Warnf("Unexpected error returned from request.Cookie(%s): %v", stateCookieName, err)
```

Comment:
Can we change this to better describe the SG functional scenario?  i.e. "Unexpected error attempting to read OIDC state cookie: %v"?